File: week-1/day-2/functions/unique/tests/TestUnique.java
import org.junit.Test;
import static org.junit.Assert.*;

public class TestUnique {

  @Test
  public void testHappyCase() {
    assertArrayEquals(new int[]{1, 11, 34, 52, 61}, 
      Unique.unique(new int[]{1, 11, 34, 11, 52, 61, 1, 34}));
   }
}

File: week-1/day-2/functions/unique/tests/test_unique.py
import unittest
from unique import unique


class Unique(unittest.TestCase):

    def test_unique(self):
        self.assertEqual([1, 11, 34, 52, 61],
                         unique([1, 11, 34, 11, 52, 61, 1, 34]))

if __name__ == '__main__':
    unittest.main()
